themes/default/scripts/register.js

```diff
@@ -414,4 +414,30 @@ function onCheckChange()
 	}
 	else
 		document.forms.postForm.emailPassword.disabled = false;
-}
\ No newline at end of file
+}
+
+(function($) {
+	$(document).ready(function() {
+		$('#agreement_lang').change(function() {
+			$.ajax({
+				type: "POST",
+				url: elk_scripturl + "?action=register;xml;api=json;",
+				data: {sa: 'agrelang', lang: $(this).val()},
+				beforeSend: ajax_indicator(true)
+			})
+			.done(function(request) {
+				if (request != '')
```

Comment:
hmm... I don't **think** request can be not set and it should at least be a string (though jquery documentation is not exactly easy to read).